[2025-04-28T06:45:27.647+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:45:27.658+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:45:27.659+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-28T06:45:27.674+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-04-28T06:45:27.685+0000] {standard_task_runner.py:57} INFO - Started process 368 to run task
[2025-04-28T06:45:27.692+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmplgdno2tj']
[2025-04-28T06:45:27.703+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask insert_viruses
[2025-04-28T06:45:27.824+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host 77b690a9fde8
[2025-04-28T06:45:27.998+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-04-28T06:45:28.095+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-04-28T06:45:28.096+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-28T06:45:28.109+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250428T064527, end_date=20250428T064528
[2025-04-28T06:45:28.153+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-28T06:45:28.179+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-03T07:22:09.828+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-03T07:22:09.829+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-03T07:22:09.868+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-03T07:22:09.868+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-03T07:22:09.912+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-03T07:22:09.922+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-03T07:22:09.922+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-03T07:22:09.936+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-05-03T07:22:09.942+0000] {standard_task_runner.py:57} INFO - Started process 253 to run task
[2025-05-03T07:22:09.946+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp4h2yed10']
[2025-05-03T07:22:09.949+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask insert_viruses
[2025-05-03T07:22:09.998+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-03T07:22:09.999+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-03T07:22:10.000+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host f2b491f5d506
[2025-05-03T07:22:10.076+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-03T07:22:10.131+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-05-03T07:22:10.132+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-03T07:22:10.141+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250503T072209, end_date=20250503T072210
[2025-05-03T07:22:10.199+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-03T07:22:10.225+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-05T13:47:40.234+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-05T13:47:40.235+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-05T13:47:40.272+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-05T13:47:40.272+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-05T13:47:40.463+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-05T13:47:40.475+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-05T13:47:40.476+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-05T13:47:40.491+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-05-05T13:47:40.497+0000] {standard_task_runner.py:57} INFO - Started process 209 to run task
[2025-05-05T13:47:40.501+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpvihpmvqg']
[2025-05-05T13:47:40.503+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask insert_viruses
[2025-05-05T13:47:40.550+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-05T13:47:40.551+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-05T13:47:40.552+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host 1cb5a07b27d6
[2025-05-05T13:47:40.632+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-05T13:47:40.688+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-05-05T13:47:40.689+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-05T13:47:40.702+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250505T134740, end_date=20250505T134740
[2025-05-05T13:47:40.753+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-05T13:47:40.777+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-10T17:03:05.345+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-10T17:03:05.346+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-10T17:03:05.390+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-10T17:03:05.390+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-10T17:03:05.443+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-10T17:03:05.456+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-10T17:03:05.457+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-10T17:03:05.514+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-05-10T17:03:05.529+0000] {standard_task_runner.py:57} INFO - Started process 522 to run task
[2025-05-10T17:03:05.532+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmprxhe7a1t']
[2025-05-10T17:03:05.538+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask insert_viruses
[2025-05-10T17:03:05.612+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-10T17:03:05.652+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-10T17:03:05.654+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host 45059e99c3e8
[2025-05-10T17:03:05.763+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-10T17:03:05.833+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-05-10T17:03:05.834+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-10T17:03:05.846+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250510T170305, end_date=20250510T170305
[2025-05-10T17:03:05.906+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-10T17:03:05.932+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-11T16:06:01.367+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-11T16:06:01.368+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-11T16:06:01.412+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-11T16:06:01.413+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-11T16:06:01.458+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-11T16:06:01.470+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-11T16:06:01.471+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-11T16:06:01.486+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-05-11T16:06:01.495+0000] {standard_task_runner.py:57} INFO - Started process 269 to run task
[2025-05-11T16:06:01.508+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp1_tq5qvy']
[2025-05-11T16:06:01.512+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask insert_viruses
[2025-05-11T16:06:01.570+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-11T16:06:01.571+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-11T16:06:01.573+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host 9a7df5060863
[2025-05-11T16:06:01.678+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-11T16:06:01.734+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-05-11T16:06:01.735+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-11T16:06:01.745+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250511T160601, end_date=20250511T160601
[2025-05-11T16:06:01.801+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-11T16:06:01.823+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-12T09:21:26.446+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-12T09:21:26.447+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-12T09:21:26.483+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-12T09:21:26.484+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-12T09:21:26.742+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-12T09:21:26.754+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-12T09:21:26.755+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-12T09:21:26.775+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-05-12T09:21:26.783+0000] {standard_task_runner.py:57} INFO - Started process 236 to run task
[2025-05-12T09:21:26.791+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpx0qcjmw_']
[2025-05-12T09:21:26.796+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask insert_viruses
[2025-05-12T09:21:26.879+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-12T09:21:26.880+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-12T09:21:26.882+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host 16fa90749134
[2025-05-12T09:21:26.977+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-12T09:21:27.027+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-05-12T09:21:27.028+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-12T09:21:27.043+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250512T092126, end_date=20250512T092127
[2025-05-12T09:21:27.084+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-12T09:21:27.107+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-13T12:50:06.368+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-13T12:50:06.369+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-13T12:50:06.415+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-13T12:50:06.416+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-13T12:50:06.472+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-13T12:50:06.486+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-13T12:50:06.487+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-13T12:50:06.506+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-05-13T12:50:06.517+0000] {standard_task_runner.py:57} INFO - Started process 654 to run task
[2025-05-13T12:50:06.521+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp4dcmkqul']
[2025-05-13T12:50:06.524+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask insert_viruses
[2025-05-13T12:50:06.605+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-05-13T12:50:06.606+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-05-13T12:50:06.609+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host 1d46261ea2e6
[2025-05-13T12:50:06.705+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-13T12:50:06.772+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-05-13T12:50:06.773+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-13T12:50:06.787+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250513T125006, end_date=20250513T125006
[2025-05-13T12:50:06.854+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-13T12:50:06.891+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-07T14:27:19.909+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-06-07T14:27:19.910+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-06-07T14:27:20.018+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-06-07T14:27:20.020+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-06-07T14:27:20.121+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-06-07T14:27:20.139+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-06-07T14:27:20.140+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-06-07T14:27:20.175+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-06-07T14:27:20.190+0000] {standard_task_runner.py:57} INFO - Started process 231 to run task
[2025-06-07T14:27:20.195+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp7vo4bgs7']
[2025-06-07T14:27:20.200+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask insert_viruses
[2025-06-07T14:27:20.332+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-06-07T14:27:20.333+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-06-07T14:27:20.336+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host fad517f3b141
[2025-06-07T14:27:20.492+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-06-07T14:27:20.873+0000] {logging_mixin.py:151} INFO - Table utilisée: pandemics.virus
[2025-06-07T14:27:20.893+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-06-07T14:27:20.894+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-06-07T14:27:20.910+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250607T142720, end_date=20250607T142720
[2025-06-07T14:27:20.974+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-07T14:27:21.013+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-23T07:34:48.700+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-06-23T07:34:48.701+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-06-23T07:34:48.727+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-06-23T07:34:48.727+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-06-23T07:34:48.761+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-06-23T07:34:48.771+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-06-23T07:34:48.772+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-06-23T07:34:48.785+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_viruses> on 2025-03-10 00:00:00+00:00
[2025-06-23T07:34:48.792+0000] {standard_task_runner.py:57} INFO - Started process 269 to run task
[2025-06-23T07:34:48.795+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_viruses', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp8dktgo2f']
[2025-06-23T07:34:48.797+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask insert_viruses
[2025-06-23T07:34:48.844+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509
[2025-06-23T07:34:48.844+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_viruses'
[2025-06-23T07:34:48.846+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_viruses scheduled__2025-03-10T00:00:00+00:00 [running]> on host ac5b3d82d6a8
[2025-06-23T07:34:48.924+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_viruses' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-06-23T07:34:49.153+0000] {logging_mixin.py:151} INFO - Table utilisée: pandemics.virus
[2025-06-23T07:34:49.159+0000] {logging_mixin.py:151} INFO - Insertion des virus terminée.
[2025-06-23T07:34:49.160+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-06-23T07:34:49.170+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_viruses, execution_date=20250310T000000, start_date=20250623T073448, end_date=20250623T073449
[2025-06-23T07:34:49.210+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-23T07:34:49.232+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
